wct-test/json-forms.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <script src="../node_modules/webcomponents.js/webcomponents-lite.js"></script>
+    <script src="../node_modules/web-component-tester/browser.js"></script>
+    <script src="./native-shim.js"></script>
```

Comment:
can't we use the native-shim from the example or move this to some common place, so that we do not copy it multiple times?